.github/pull_request_template.md

```diff
@@ -1,20 +1,22 @@
 <!--
 If you are a first-time contributor to MontePy,
-refer the developing guidelines at:
+refer to the developing guidelines at:
 https://idaholab.github.io/MontePy/developing.html
 -->
 
 # Description
 
-Please include a summary of the change and which issue is fixed if applicable. Please also include relevant motivation and context.
+Please include a summary of the change and which issue is fixed, if applicable. Please also include relevant motivation and context.
 
 Fixes # (issue)
 
 # Checklist
 
-- [ ] I have performed a self-review of my own code
-- [ ] I have made corresponding changes to the documentation (if applicable)
-- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
+- [ ] I have performed a self-review of my own code.
+- [ ] The code follows the standards outlined in the [development documentation](https://idaholab.github.io/MontePy/developing.html).
+- [ ] I have made corresponding changes to the documentation, providing clear details on the added or modified functionality (if applicable).
+- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable).
+- [ ] I have checked that my code achieves the required test coverage, and I have included coverage reports (if applicable).
```

Comment:
I'm realizing this is hacky solution right now. I am working to debug why coveralls isn't sending status updates. 
I just want the merger (usually me) to add a link to the coveralls report. A link to coveralls: [https://coveralls.io/github/idaholab/MontePy/](https://coveralls.io/github/idaholab/MontePy/) should be included to be able to quickly get there.

Reply:
We actually fixed coveralls. So you can remove the reference to coverage reports.